### DISCUSSION TITLE
Fonts stored locally

### DISCUSSION THREAD

**Discussion by Zagrthos** [AzuraCast/AzuraCast#6598] [General]
Hey everybody,

thanks for making such a great software! We looked for something like this and we're overwhelmed by the easiness.

Now to my request:
First of all, we use the docker method to install AzuraCast and I'm using Firefox to connect to the web interface.

I've did some "tests" to look if I can improve the NGINX webserver behind and while looking through the data in the browser I found out that you use some google fonts which are taken by an URL. Most sites these days offer the ability to store all fonts locally so maybe that could be an option for you too? Or are there some other methods to store them locally inside the docker container which I already can utilize? I've attached a pic.

![image](https://github.com/AzuraCast/AzuraCast/assets/36556009/cc318353-b9ab-47f3-b9ba-dececdcadbae)

Here in Germany we (sadly) already have some courts which think connections to google servers (while using they're fonts) are not that great and could be easily sued because of data-privacy.

**Comment by BusterNeece** [comment]
@Zagrthos We actually *do* have all of our fonts completely self-contained on our installations. All of our assets are built and managed within the installation, so in theory you don't even need an outside Internet connection for AzuraCast to run properly.

I looked through our SCSS files and we don't reference external URLs at any point in the build process. I'm not sure what version you're using in this case, but it certainly shouldn't be an issue in the latest Stable or Rolling Release versions.

**Reply by Zagrthos** [reply]
I used the latest stable release but I can check in a few hours if the same is happening within the newest rolling Release

**Reply by Zagrthos** [reply]
Okay I take it all back. This is fixed in the latest rolling release version.

Sorry for the inconvenience!
